src/seeds.proposals.cpp

```diff
@@ -528,8 +528,13 @@ void proposals::evalproposal (uint64_t proposal_id, uint64_t prop_cycle) {
       }
 
     } else {
+
+      asset stackToBurn = valid_quorum ?  pitr->staked : asset(pitr->staked.amount * 0.05, seeds_symbol);
```

Comment:
typo - should be "stakeToBurn" - please fix

Reply:
Code looks great.

Reply:
Oh, my bad, fixing it!